### ISSUE TITLE
EmberObject returns 404

### ISSUE THREAD

**Issue by benmurden** [ember-learn/ember-api-docs#704]
https://api.emberjs.com/ember/release

This page has a link to EmberObject that points to the following URL

https://api.emberjs.com/ember/release/classes/EmberObject

however, this returns a 404 error.

**Comment by locks** [comment]
I cannot reproduce 🤔

**Comment by Gaurav0** [comment]
Tried throttling to slow 3G and hard reloading. Still unable to reproduce. :(

**Comment by Gaurav0** [comment]
OK. Can reproduce by blocking domain https://api-store.emberjs.com

I'll assume the user was not able to connect to the domain but that fastboot was working, so it looks like all is well until he clicks EmberObject.

@sivakumar-kailasam @locks @toddjordan What sort of "more correct" error message should we show, or should we leave the 404 alone?

**Comment by benmurden** [comment]
This was actually more limited in scope - just this link (that I had tried) was not working. I had been all around various Enber and Ember Data APIs before encountering this one. I actually first encountered it while trying to get to EmberObject from an inherits reference.

Seems to be resolved on an incognito window, so maybe a cache issue.

**Comment by Gaurav0** [comment]
Ok, these issues can obviously have multiple causes.

@benmurden You may have experienced a problem that could have been fixed by the solution @alexeykostevich provided in #672

**Comment by benmurden** [comment]
Yes, the steps in https://github.com/ember-learn/ember-api-docs/issues/701#issuecomment-617232427 reproduce the issue for me.

**Comment by Gaurav0** [comment]
@benmurden Is the issue fixed for you?

**Comment by benmurden** [comment]
@Gaurav0 The issue is fixed for Ember project links, but if I switch to Ember Data under Projects, all the links on the right return 404 with console errors like the following.
```
TypeError: Cannot read property 'ember-data-3.17.0-EmberObject' of undefined
```
